### PR TITLE
include component for sequence end

### PR DESCRIPTION
**Pull request by jwong-d2l**

### CODE REVIEW

**Comment by benyamin-noori** [comment]
I don't understand where this component is going to be used. Are we going to replace the end-of-lesson holder element with this? Or is this the page the user sees when they finish the sequence? 

Don't we have a eol-main component in sequences for that?

**Comment by jwong-d2l** [comment]
This is the component that displays the end-of-lesson menu item in the navigator.

File: components/d2l-sequence-end.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+					_containerClass:{
+						type: String,
+						computed: '_getContainerClass(currentActivity)'
+					},
+					currentActivity: {
+						type: String,
+						reflectToAttribute: true,
+						notify: true
+					},
+					text:{
```
Comment (lawong-d2l) [inline components/d2l-sequence-end.html]: place holder for now I guess?

Comment (jwong-d2l) [reply components/d2l-sequence-end.html]: no, it gets set by the parent component.

**Pull request merged**